connection/user.feature

```diff
@@ -36,6 +36,14 @@ Feature: Connection Users
     Given users delete: user
     Then users not contains: user
 
+  Scenario: users can get themselves
```

Comment:
I think this is not really in our 'style' grammatically? Bit informal - try calling this 'connected user is retrievable' and call the step 'Then get connected user' instead of 'user get self'?

Reply:
Agreed, waiting on CI.